src/visualizers/widgets/ArtifactIndex/ModalControl.js

```diff
@@ -0,0 +1,31 @@
+/*globals define, $*/
+define([
+    'deepforge/viz/Utils',
+    'text!./ArtifactModal.html'
+], function(
+    Utils,
+    MODAL_HTML
+) {
+    'use strict';
+
+    const ModalControl = function() {
```

Comment:
Out of curiousity, why is this called `ModalControl`? It's name is very similar to the naming convention for controllers in `src/visualizers/panels/<visualizer>/` which could be a little confusing.

Given that the actual html file is called `ArtifactModal.html`, I would've expected it to be called something like `ArtifactModal,js`. Otherwise, something like `ArtifactInfo`or `ArtifactInfo{Dialog,Modal}.js` also might make sense.

Reply:
This was named `ModalControl` because it controls the display of the modal.